pkg/k8shandler/configmaps.go

```diff
@@ -46,6 +46,53 @@ type indexSettingsStruct struct {
 	ReplicaShards string
 }
 
+// CreateOrUpdateConfigMap reconciles a configmap
+func (elasticsearchRequest *ElasticsearchRequest) CreateOrUpdateConfigMap(cm *v1.ConfigMap) error {
+	dpl := elasticsearchRequest.cluster
+	err := elasticsearchRequest.client.Create(context.TODO(), cm)
+	if err == nil {
+		logrus.Info("created configmap", "name", cm.Name)
+		return nil
+	}
+	if !errors.IsAlreadyExists(err) {
+		return fmt.Errorf("failure constructing ConfigMap: %w", err)
```

Comment:
since we have a bunch of different formatters, what is `%w` doing vs `%v` that we've used in other places?

Reply:
%v is a sledgehammer that shouldn't really be used that much IMO. 
%w is the go standard for error wrapping. It's used with errors.Unwrap, etc.